src/Servers/Kestrel/Core/ref/Microsoft.AspNetCore.Server.Kestrel.Core.netcoreapp3.0.cs

```diff
@@ -160,24 +159,6 @@ public partial class MinDataRate
         public System.TimeSpan GracePeriod { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } }
     }
 }
-namespace Microsoft.AspNetCore.Server.Kestrel.Core.Adapter.Internal
```

Comment:
@sebastienros This will break the benchmarks.

Reply:
Specifically this https://github.com/aspnet/Benchmarks/blob/fd69bc294bdfda8d23a9e094dfbbb4a56fa82651/src/Benchmarks/PassthroughConnectionFilter.cs#L12